loadtest/sanitycheck/sanity_check_register_validator.go

```diff
@@ -52,9 +52,8 @@ func (t *RegisterValidatorTest) Run() error {
 	fmt.Println("Running", t.Name())
 	defer fmt.Println("Finished", t.Name())
 
-	_, err := t.runTest()
-
-	return err
+	// _, err := t.runTest()
```

Comment:
Add a warning using `fmt` that you are skipping this test. Also leave a TODO, when you actually implement registration, this sanity check is really helpful on test nets.